pkg/metrics/otel.go

```diff
@@ -14,6 +14,10 @@
 
 package metrics
 
+import (
+	"gopkg.in/yaml.v3"
```

Comment:
This package is archived, use `sigs.k8s.io/yaml` instead.

Reply:
Fixed